app/controllers/concerns/blacklight/search_fields.rb

```diff
@@ -53,6 +57,7 @@ def label_for_search_field(key)
        field_def.label
```

Comment:
thanks @jcoyne. Here is an example of a run where I did that: https://travis-ci.org/projectblacklight/blacklight/jobs/306000135. I'm not sure what causes the problem with blacklight_config but I sort of figured it had to do with calling the helper through the controller so I backed off. If you can help advise me on how to resolve all those failures I'm could do it. Otherwise as I mentioned in my previous comment I think the direction to the new method is probably fine.

Reply:
@jcoyne here's the commit that goes with that run: https://github.com/projectblacklight/blacklight/pull/1811/commits/1a40e86160e25c7a55afa9979f7a84c21146d922

how big of a deal-breaker is this?

Reply:
@cbeer can you weigh in here?

Reply:
Sorry, just getting to this. I didn't realize the old and new method names were the same, so it makes sense you'll have a bad time trying to deprecate the old name. In #1815, I've taken your commits, removed the old method, preserved the existing tests for that method, and also backports a couple bug fixes that were introduced after 57c6bbf.

Reply:
Thank you @cbeer !!!